src/explore-education-statistics-common/src/utils/locationLevelsMap.ts

```diff
@@ -0,0 +1,91 @@
+import { Dictionary } from '@common/types';
+
+const locationLevelsMap: Dictionary<Dictionary<string>> = {
```

Comment:
Type would be better as:

```ts
Dictionary<{
  label: string;
  plural: string;
  prefix: string
}>
```

This is safer and should provide autocomplete

Reply:
Done